### ISSUE TITLE
lambda parameter

### ISSUE THREAD

**Issue by mynameisvinn** [MorvanZhou/Reinforcement-learning-with-tensorflow#26]
https://github.com/MorvanZhou/Reinforcement-learning-with-tensorflow/blob/639902bdd8109e115b1e22575b8f9b467e09f863/contents/8_Actor_Critic_Advantage/AC_CartPole.py#L165

do you think we should specify lambda hyperparameter, rather than hardcoding it to 0.95?

**Comment by MorvanZhou** [comment]
0.95 in line 165 is not lambda, instead, it is just a smothing factor for presenting reward
